[CHG] partner_contact_in_several_companies

valentinthirion opened the pull request:
Improved view for better clarity of the fonctions for a partner

valentinthirion committed:
[CHG] partner_contact_in_several_companies
Improved view for better clarity of the fonctions for a partner

yajo reviewed (approved):
Code OK

valentinthirion committed:
[CHG] partner_contact_in_several_companies
Added image and tags in the function view

valentinthirion committed:
[CHG] partner_contact_in_several_companies
Improved the view

MiquelRForgeFlow commented on partner_contact_in_several_companies/views/res_partner.xml:
```diff
@@ -114,6 +118,12 @@
+                                <div>
+                                    <group col="2">
+                                        <field name="image" widget="image" options="{'preview_image': 'image_medium', 'size': [90, 90]}" modifiers="{}"/>
+                                        <field name="category_id" widget="many2many_tags" placeholder="Étiquettes..." can_create="true" can_write="true" modifiers="{}"/>
```
shouldn't be `tags` or `labels` instead of `Étiquettes`?

valentinthirion committed:
[FIX] partner_contact_in_several_companies
Corrected 'Etiquettes' to 'Tags'

valentinthirion committed:
[CHG] partner_contact_in_several_companies
Allowed to create a position and set the company it is linked to

valentinthirion committed:
[ADD] partner_contact_in_several_companies
Edited the contact view to be able to set picture and tags from contact pop-up form in contact form

ghost commented:
:+1: LGTM

fwyaime committed:
[ADD] button for partner related documents in partner form view

fwyaime committed:
[RMV] push on wrong branch

fwyaime committed:
[ADD] button for partner related documents in partner form view

fwyaime committed:
[ADD] Nb of documents related + action name

valentinthirion committed:
Merge pull request #7 from abakus-it/9.0_FW_partner-related-docs

[ADD] button for partner related documents in partner form view

nikul-serpentcs commented on partner_related_documents/__init__.py:
```diff
@@ -0,0 +1 @@
+from . import models
```
@valentinthirion Add New Blank Line

nikul-serpentcs commented on partner_related_documents/__openerp__.py:
```diff
@@ -0,0 +1,21 @@
+        "website_sign",
+    ],
+    'author': "François Wyaime, AbAKUS it-solutions SARL",
+    'website': "http://www.abakusitsolutions.eu",
```
URL Should be `https://github.com/OCA/partner-contact`

nikul-serpentcs commented on partner_related_documents/__openerp__.py:
```diff
@@ -0,0 +1,21 @@
+    'depends': [
+        "website_sign",
+    ],
+    'author': "François Wyaime, AbAKUS it-solutions SARL",
```
Added OCA in author list

nikul-serpentcs reviewed (changes requested):
Improve Code

valentinthirion committed:
[FIX] Fixed non viewable form view

NL66278 commented:
@valentinthirion Will you look into solving the runbot and travis failures?

NL66278 commented:
@valentinthirion Please fix at least these:
```
partner_related_documents/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

************* Module partner_related_documents.__openerp__

partner_related_documents/__openerp__.py:1: [C8201(no-utf8-coding-comment), ] No UTF-8 coding comment found: Use `# coding: utf-8` or `# -*- coding: utf-8 -*-`

partner_related_documents/__openerp__.py:1: [C8101(manifest-required-author), ] One of the following authors must be present in manifest: 'Odoo Community Association (OCA)'

partner_related_documents/__openerp__.py:1: [C8102(manifest-required-key), ] Missing required key "license" in manifest file

partner_related_documents/__openerp__.py:1: [C8103(manifest-deprecated-key), ] Deprecated key "description" in manifest file

partner_related_documents/__openerp__.py:1: [C8106(manifest-version-format), ] Wrong Version Format "9.0.1.0" in manifest file. Regex to match: "(9\.0)\.\d+\.\d+\.\d+$"

************* Module partner_contact_in_several_companies

partner_contact_in_several_companies/views/res_partner.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

partner_contact_in_several_companies/demo/ir_actions.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

partner_contact_in_several_companies/demo/res_partner.xml:1: [W7908(missing-newline-extrafiles), ]  Missing newline

************* Module partner_contact_in_several_companies.models.res_partner

partner_contact_in_several_companies/models/res_partner.py:15: [C8108(method-compute), ResPartner] Name of compute method should start with "_compute_"

partner_contact_in_several_companies/models/res_partner.py:28: [W8104(api-one-deprecated), ResPartner._get_contact_type] api.one deprecated

************* Module partner_contact_in_several_companies.models.ir_actions

partner_contact_in_several_companies/models/ir_actions.py:13: [R8110(old-api7-method-defined), IRActionsWindow.read] Method defined with old api version 7
```

NL66278 commented:
No reaction on change requests or questions.
